scripts/gaussian_codegen/cdf/derive.py

```
@@ -0,0 +1,242 @@
+
+    print()
+    print(f"Chosen: degree {degree} (n_coeffs = {degree + 1}), max error {err:.3e} at z={worst_z:.4f}")
+
+    # Coefficient magnitude check at WAD scale before serialization.
+    wad = mpf(constants.WAD)
+    max_mag = mpf(0)
+    for c in num + den:
+        m = abs(c) * wad
+        if m > max_mag:
+            max_mag = m
+    print(f"Max |coeff| × WAD = {float(max_mag):.3e} (must be < 2^128 ≈ 3.4e38)")
```

Ruff (0.15.15)

[warning] String contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF001)